process/interface.go

```diff
@@ -637,3 +640,9 @@ type EpochStartDataCreator interface {
 	VerifyEpochStartDataForMetablock(metaBlock *block.MetaBlock) error
 	IsInterfaceNil() bool
 }
+
+// FinalityAttester is able to manage the final blocks
+type FinalityAttester interface {
```

Comment:
Maybe rename FinalityAttester to something nice :).

Reply:
Do you have a suggestion? FinalityAttester describes best what this interface should suppose to do: attest finality (in case of block headers)

Reply:
HeaderFinalityAttester? @AdoAdoAdo